src/main/java/org/jitsi/videobridge/octo/OctoTentacle.java

```diff
@@ -312,8 +312,9 @@ public void addRtpExtension(RtpExtension rtpExtension)
      */
     public void expire()
     {
+        logger.info("Expiring");
```

Comment:
This message is a bit terse.  Add what you're expiring?

Reply:
it's terse just because the log context will already give the other info

Reply:
I forget, does log context show up even if someone has the "wrong" log formatter configured?  I forget the order of operations.

Reply:
no, that is done in the formatter, but we've already done this in some other places and, although i want to make sure things don't 'break' with old formatter i'm not really feeling like fully-accommodating it either, as we should just be migrating.  more motivation :)

Reply:
Fair enough.